Dockerfile

```diff
@@ -2,8 +2,6 @@ FROM node:12.14-alpine
 
 WORKDIR /app
 
-ENV CDN_PRODUCTION_URL=https://d1s2w0upia4e9w.cloudfront.net CDN_STAGING_URL=https://d1rmpw1xlv9rxa.cloudfront.net
```

Comment:
This syntax is incorrect, you shouldn't be using an `=` -- see an example here: https://github.com/artsy/volt/blob/master/Dockerfile#L3

Reply:
Actually I see this listed both ways in the file - here are the env docs, happy to pair on getting this working, https://docs.docker.com/engine/reference/builder/#environment-replacement

Reply:
Interesting that this ever worked on staging?

Reply:
That is weird, it worked on staging but not prod?

Reply:
I pulled that syntax directly from the docs

> The second form, ENV <key>=<value> ..., allows for multiple variables to be set at one time. Notice that the second form uses the equals sign (=) in the syntax, while the first form does not. Like command line parsing, quotes and backslashes can be used to include spaces within values.

https://docs.docker.com/engine/reference/builder/#env

Reply:
Seemed to: 

<img width="610" alt="Screen Shot 2020-02-24 at 10 00 07 AM" src="https://user-images.githubusercontent.com/236943/75178178-79906c00-56ec-11ea-9d7a-0ff0fb4bc7a9.png">

Reply:
Ah, let me do hard cache clear